sm/plugin_sm/tests/plugin.rs

```diff
@@ -195,13 +195,7 @@ mod tests {
         let res = plugin.check_module_type(&module);
 
         // Plugin with type None is not registered and expected plugin to use would be `default`.
```

Comment:
This comment is puzzling.

Do you mean:
```
A module with type None can be processed by any module.
```

Reply:
Yes. I'll update the comment.